bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
Thanks for the actual info you used to find the mismatch. Very useful.

If you go look in my actual applied patch I did what I suggested you could do - I do not *just* print out the approximated bz-query.